src/order/heyting/basic.lean

```diff
@@ -442,6 +445,10 @@ by rw [sup_sdiff, sdiff_self, bot_sup_eq, h.sdiff_eq_right]
 lemma disjoint.sup_sdiff_cancel_right (h : disjoint a b) : (a ⊔ b) \ b = a :=
 by rw [sup_sdiff, sdiff_self, sup_bot_eq, h.sdiff_eq_left]
 
+/-- See `le_sdiff` for a stronger version in generalised Boolean algebras. -/
```

Comment:
Does `codisjoint.himp_le_of_right_le` not also have a stronger version?

Reply:
Stronger version added.